_install-and-configure/configuring-opensearch/index-settings.md

```diff
@@ -143,6 +143,8 @@ OpenSearch supports the following dynamic index-level index settings:
 
 - `index.search.idle.after` (Time unit): The amount of time a shard should wait for a search or get request until it goes idle. Default is `30s`.
 
+- `index.search.default_pipeline` (String): The default search pipeline for the index to be used on each search request. If the default pipeline is set and the pipeline does not exist, then index requests fail. The pipeline name _none specifies that the index does not have an search pipeline.
+
```

Comment:
We may want to incorporate the changes (and the limitations) from https://github.com/opensearch-project/documentation-website/pull/7053/files too.

Prior to OpenSearch 2.14, `index.search.default_pipeline` would only work if you search exactly 1 index using its name. If you tried to us an index pattern, an alias, or multiple indices, then no default pipeline would be applied.

With the linked change, we resolve index names, aliases, patterns to the real indices, check to see which default search pipelines they use (if any). If all resolved indices use the same pipeline, we'll use it. Otherwise, we still won't apply a search pipeline.

Reply:
Thanks, @msfroh, I just relabeled #7053 for 2.14.

Reply:
How about we link to the updated topic in https://opensearch.org/docs/latest/search-plugins/search-pipelines/using-search-pipeline/#default-search-pipeline once 2.14 is merged?

Reply:
> How about we link to the updated topic in https://opensearch.org/docs/latest/search-plugins/search-pipelines/using-search-pipeline/#default-search-pipeline once 2.14 is merged?

I like this idea. As this new changes will make the text way more lengthy